sdk/servicebus/azure-servicebus/tests/servicebus_preparer.py

```diff
@@ -255,6 +255,7 @@ def __init__(self,
                  requires_duplicate_detection=False,
                  dead_lettering_on_message_expiration=False,
                  requires_session=False,
+                 lock_duration='PT30S',
```

Comment:
if this var is able to be modified now, add it to the set_cache stanza (since it's part of what makes this service bus queue instance unique in the cache)